docs/streams/upgrade-guide.html

```diff
@@ -688,7 +688,7 @@ <h3><a id="streams_api_changes_100" href="#streams_api_changes_100">Streams API
         If you are monitoring on task level or processor-node / state store level Streams metrics, please note that the metrics sensor name and hierarchy was changed:
         The task ids, store names and processor names are no longer in the sensor metrics names, but instead are added as tags of the sensors to achieve consistent metrics hierarchy.
         As a result you may need to make corresponding code changes on your metrics reporting and monitoring tools when upgrading to 1.0.0.
-        Detailed metrics sensor can be found in the <a href="#kafka_streams_monitoring">Streams Monitoring</a> section.
+        Detailed metrics sensor can be found in the <a href="/{{version}}/documentation/#kafka_streams_monitoring">Streams Monitoring</a> section.
```

Comment:
`#kafka_streams_monitoring` is located in `/documentation`, not the current path (i.e. `documentation/streams/upgrade-guide`)